_layouts/home.html

```diff
@@ -264,6 +264,17 @@ <h3>Legend</h3>
      * @returns {string}
      */
     function popupTemplate(shelter) {
+
+        // These seem to come in as proper `null` when manually tested
+        if (shelter.address === 'null') { shelter.address = null }
+        if (shelter.pets === 'null') { shelter.pets = null }
+        if (shelter.notes === 'null') { shelter.notes = null }
+
+        let petsExtraNotes = shelter.pets_notes;
+        if (!!petsExtraNotes === 'null' || petsExtraNotes) {
```

Comment:
I don't think this condition always does what you expect. If I understand correctly, you'd like to test whether the value of `petsExtraNotes` is `null` or `'null'`, and if so, set the value to `''`. When I take a look at the return values in console, it looks something like this:

```
let petsExtraNotes = null;
console.log(!!petsExtraNotes === 'null' || petsExtraNotes)
// => null
// (should be true)
console.log((!!petsExtraNotes === 'null'))
// => false
// (correct)
console.log(petsExtraNotes)
// => null
// (should be true when it's null, so this is likely the problem)
console.log(false || null)
// => null
petsExtraNotes = 'null';
console.log(!!petsExtraNotes === 'null' || petsExtraNotes)
// => "null"
// (should be true/truthy, so OK)
console.log(!!petsExtraNotes === 'null')
// => false
console.log(petsExtraNotes)
// => "null"
console.log(!!petsExtraNotes === 'null')
// => false
```

I think probably something more like this is needed here:
```
petsExtraNotes = null;
console.log(!petsExtraNotes || petsExtraNotes === 'null')
// => true
petsExtraNotes = 'null';
console.log(!petsExtraNotes || petsExtraNotes === 'null')
// => true
```

Reply:
Dang, that was for a whole thing for a console log test that I thought I'd cleaned up. Good catch!

Reply:
Happy to help. Thanks for your work on this bug!